langfilter/langfilter.php

```diff
@@ -178,6 +178,6 @@ function langfilter_prepare_body(App $a, &$b)
 
 	if (!$spoken) {
 		$rnd = random_string(8);
-		$b['html'] = '<div id="langfilter-wrap-' . $rnd . '" class="fakelink" onclick=openClose(\'langfilter-' . $rnd . '\'); >' . L10n::t('unspoken language %s - Click to open/close', $lang) . '</div><div id="langfilter-' . $rnd . '" style="display: none; " >' . $b['html'] . '</div>';
```

Comment:
Why that change? I think it is a rather important information that the posting was hidden because of the language detected.